tests/edge-cases/list-04.txt

```diff
@@ -0,0 +1,2 @@
+brun '(l ())'
```

Comment:
This is a good one. I mention this in the doc, but I'll try to mention it several times.